docs/using_custom_header.rst

```diff
@@ -0,0 +1,119 @@
+.. title:: Custom Header Component Documentation
+
+Custom Header Component
+=======================
+
+Overview
+--------
+
+The ``Header`` component is used to display a header with a provided ``logo``, ``mainMenuItems``,
+``secondaryMenuItems``, and ``userMenuItems`` props. If props are provided, the component will use them; otherwise,
+If any of the props ``(logo, mainMenuItems, secondaryMenuItems, userMenuItems)`` are not provided, default
+items are displayed. This component provides flexibility in customization, making it suitable for a wide
+range of applications.
+
+Props Details
+-------------
+
+The `Header` component accepts the following **optional** props for customization:
+
+``logo``
+*******
+
+The logo prop is an object containing `src`, `alt`, and `href` properties. If not passed, LOGO_URL from config will be used.
```

Comment:
I am wondering if we can get rid of this prop? Now that I am reviewing this, I am wondering that if anyone needs to use a custom logo, they can simply override the `LOGO_URL` in their MFE config. What are your thoughts on this?

Reply:
Thats correct. We can definitely remove it.